pkgs/by-name/ni/nix-editor/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "nix-editor";
+  version = "0.3.0-unstable-2023-12-20";
+
+  src = fetchFromGitHub {
+    owner = "snowfallorg";
+    repo = "nix-editor";
+    rev = "b5017f8d61753ce6a3a1a2aa7e474d59146a8ae3";
+    hash = "sha256-Ne9NG7x45a8aJyAN+yYWbr/6mQHBVVkwZZ72EZHHRqw=";
+  };
+
+  cargoHash = "sha256-ZWSLW+mFT+Qls2GeVPamispL8ObieXT4tDc1Go5cjL4=";
```

```suggestion
  cargoHash = "sha256-1OErQ+bKKInz7pQQCKW82BGODzbD9EFImBnwbiojJgk=";
```

seems like there's a hash mismatch here when I build locally

---

Or I wonder is this #308089 rearing its head?

---

No, I get the same hash mismatch on Linux.